src/Illuminate/Broadcasting/BroadcastController.php

```diff
@@ -22,4 +22,20 @@ public function authenticate(Request $request)
 
         return Broadcast::auth($request);
     }
+
+    /**
+     * Authenticate the request for user authorization.
+     * See: https://pusher.com/docs/channels/server_api/authenticating-users/#user-authentication.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function authenticateUser(Request $request)
+    {
+        if ($request->hasSession()) {
+            $request->session()->reflash();
```

Comment:
Can you explain the purpose of this?

Reply:
It's explained here: https://github.com/laravel/framework/pull/27647